src/test/java/log_in/LogInTest.java

```diff
@@ -22,7 +22,7 @@ public class LogInTest {
     LogInInputBoundary interactor;
     LogInOutputBoundary presenter;
     LogInController controller;
-    LogInUserFactory userFactory;
+    FavouriteUserFactory userFactory;
```

Comment:
We'll have to fix this so it works with the LogInUserFactory instead, as that is the object being used in the log in use case